Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2018"
 arrayvec = { version = "0.7", default-features = false }
 serde = { version = "1.0.102", optional = true }
 parity-scale-codec-derive = { path = "derive", version = "2.2.0", default-features = false, optional = true }
-bitvec = { version = "0.20.1", default-features = false, features = ["alloc"], optional = true }
+bitvec = { version = "0.22.3", default-features = false, features = ["alloc"], optional = true }
```

Comment:
what is the reason for the version bump ?

changing bitvec version to a non compatible one will require us to release a new major version of scale-codec, can't we stay on bitvec 0.20 ?

Reply:
That was an attempt to solve the miri error. I was hoping that a version bump would solve it, but obviously it did not, so I was stuck.

Reply:
ah I see ok